### PR TITLE
Rough English translation for network documentation. 

### PR DESCRIPTION
**Pull request by ColinNg**

### CODE REVIEW

File: en/libs/machine/network.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+This module is used to initialize various network card drivers. The network card has these function: connecting routes, disconnecting routes, viewing network card connection information, checking whether connections are made. 
+
+Please make sure the antenna is connected before using WiFi. 
+
+
+### [esp8285](#ESP8285_Module)
+
+Some development boards expose an `AT` network interface module, such as an `esp8285` interface connected to the `k210` processor via a serial port.
+
+Pin `8` is the enable pin. Instantiate a `GPIO` object set it high to enable, low to disable. To reset, set it low, wait a short while, then set it high.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]: Pin `8` is the enable pin. Instantiate a `GPIO` object and set the value to high to enable the network and low to disable it. To reset the network adapter, set it low and wait a few seconds then set it back to high.

File: en/libs/machine/network.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+There is currently a `esp32` module in the `MaixDuino` development board connected to `k210` via `SPI`. 
+There is also a separate `TF` plug-in module. 
+
+
+ESP8285_Module
+=========
+
+## network.ESP8285(uart)
+
+Instantiate an `ESP8285` NIC object, passing in an `uart` object. On the `dock` and `GO` currently supported by `MaixPy`, use the AT command module as `WiFi`. So the `uart` object is the object that communicates with the `AT` module, you can view the `uart` module routine.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
Instantiates an `ESP8285` NIC object utilizing the passed in `uart` object for communication. On the `dock` and `GO` boards, use the AT command module as `WiFi` with the `uart` object being the object that communicates with the `AT` module.
```

File: en/libs/machine/network.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+Calling this method will initialize `ESP8285`, throwing an exception if it fails.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
Calling this method will initialize communication with the `ESP8285` module, throwing an exception if it fails. 
```

File: en/libs/machine/network.md [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+Some development boards expose an `AT` network interface module, such as an `esp8285` interface connected to the `k210` processor via a serial port.
+
+Pin `8` is the enable pin. Instantiate a `GPIO` object set it high to enable, low to disable. To reset, set it low, wait a short while, then set it high. 
+
+Sample code [wifi_ap_scan.py] (https://github.com/sipeed/MaixPy_scripts/blob/master/network/demo_wifi_ap_scan.py)
+
+
+### [esp32](#ESP32_Module)
+
+There is currently a `esp32` module in the `MaixDuino` development board connected to `k210` via `SPI`.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
There is currently an `esp32` module for the `MaixDuino` development board connected to the `k210` processor via `SPI`.
```

File: en/libs/machine/network.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+## network.ESP8285(uart)
+
+Instantiate an `ESP8285` NIC object, passing in an `uart` object. On the `dock` and `GO` currently supported by `MaixPy`, use the AT command module as `WiFi`. So the `uart` object is the object that communicates with the `AT` module, you can view the `uart` module routine.
+
+Calling this method will initialize `ESP8285`, throwing an exception if it fails. 
+
+
+### Parameters
+
+* `uart`: UART object that communicates with the AT module
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
* `uart`: The UART connection to the AT module
```

File: en/libs/machine/network.md [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+### Return Value
+
+* `ESP8285`: NIC object
+
+
+## ESP8285
+
+### connect(ssid, key)
+
+Connect to a hotspot (AP/router).
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
Connect to an access point or router
```

File: en/libs/machine/network.md [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+## ESP8285
+
+### connect(ssid, key)
+
+Connect to a hotspot (AP/router). 
+
+#### Parameters
+
+* `ssid`: hotspot `SSID`
+* `key`: password for hotspot
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
* `key`: password for the network
```

File: en/libs/machine/network.md [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+
+## ESP8285
+
+### connect(ssid, key)
+
+Connect to a hotspot (AP/router). 
+
+#### Parameters
+
+* `ssid`: hotspot `SSID`
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
* `ssid`: network name
```

File: en/libs/machine/network.md [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+* `key`: password for hotspot
+
+#### Return Value
+
+None. An exception will be thrown if an error occurs.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
None. An exception will be thrown if the connection is unsuccessful.
```

File: en/libs/machine/network.md [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+nic.ifconfig()
+```
+
+#### Parameters
+
+None
+
+#### Return Value
+
+`tuple` type, the elements are all strings: `(ip, netmask, gateway, dns_server, dhcp_server, mac, ssid)`, if not queried or invalid, the value is `"0".
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
A `tuple` containing the following elements: `(ip, netmask, gateway, dns_server, dhcp_server, mac, ssid)`. If any of the values are missing or invalid, the value is `"0"`
```

File: en/libs/machine/network.md [hunk 11]
```diff
@@ -0,0 +1,267 @@
...
+
+None
+
+#### Return Value
+
+None
+
+### scan
+
+Scan for surrounding WiFi hotspots.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
Scan for surrounding WiFi networks. 
```

File: en/libs/machine/network.md [hunk 12]
```diff
@@ -0,0 +1,267 @@
...
+
+Scan for surrounding WiFi hotspots. 
+
+#### Parameters
+
+None
+
+#### Return Value
+
+A `list` object, each element containing a string, the string comes from the response of the `AT` module, the content is the same as described in `esp8285` `AT directive document`, as follows:
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
A `list` of strings  where each string represents a comma separated list of details regarding a specific network. The string comes from the response of the `AT` module where the content is the same as described in the `esp8285` `AT directive document`. An example is as follows:
```

File: en/libs/machine/network.md [hunk 13]
```diff
@@ -0,0 +1,267 @@
...
+   * 5: CIPHER_TKIP_CCMP
+   * 6: CIPHER_UNKNOWN
+* `<group_cipher>`: definition is the same as `<pairwise_cipher>`
+* `<bgn>`: bit0 stands for b mode; bit1 stands for g mode; bit2 stands for n mode
+          If the corresponding bit is 1, it indicates that the mode is enabled; if the corresponding bit is 0, the mode is not enabled.
+* `<wps>`:0, WPS is not enabled; 1, WPS is enabled
+
+example:
+```
+info_strs = ['4,"ChinaNet-lot0",-79,"c8:50:e9:e8:21:3e",1,-42,0,4,3,7,1', '4,"TOPSTEP2G4",-7
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
```python
info_strs = [
    '4,"ChinaNet-lot0",-79,"c8:50:e9:e8:21:3e",1,-42,0,4,3,7,1',
    '4,"TOPSTEP2G4",-70,"f8:e7:1e:0d:0d:f8",1,-57,0,4,4,7,0'
]
``
```

File: en/libs/machine/network.md [hunk 14]
```diff
@@ -0,0 +1,267 @@
...
+Then for example to get every `AP` `SSID`:
+
+```
+for ap_info in info:
+    print(ap_info[1])
+```
+
+### enable_ap(ssid, key, chl=5, ecn=3)
+
+Connect to the specified WiFi hotspot.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
Connect to the specified WiFi network. 
```

File: en/libs/machine/network.md [hunk 15]
```diff
@@ -0,0 +1,267 @@
...
+
+### enable_ap(ssid, key, chl=5, ecn=3)
+
+Connect to the specified WiFi hotspot. 
+
+#### Parameters
+
+* `ssid`: SSID
+* `key`: password
+* `chl`: channel number of the WiFi signal
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
* `chl`: channel number of the WiFi network
```

File: en/libs/machine/network.md [hunk 16]
```diff
@@ -0,0 +1,267 @@
...
+```
+or
+```
+Nic.enable_ap("maixpy", "12345678", 5, network.ESP8285.OPEN)
+```
+
+
+### disable_ap()
+
+Disconnect from the currently connected WiFi hotspot.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
Disconnect from the currently connected WiFi network. 
```

File: en/libs/machine/network.md [hunk 17]
```diff
@@ -0,0 +1,267 @@
...
+### disable_ap()
+
+Disconnect from the currently connected WiFi hotspot. 
+
+
+### Constants
+
+#### OPEN
+
+No encryption and no password required.
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]:
```suggestion
No encryption or password required. 
```

File: en/libs/machine/network.md [hunk 18]
```diff
@@ -0,0 +1,267 @@
...
+## network.ESP32_SPI(cs,rst,rdy,mosi,miso,sclk)
+
+To instantiate an `ESP32_SPI` NIC object, you need to pass in the corresponding `GPIOHS FUNC`.
+
+If the number of parameters passed in is incorrect, an error will be returned.
+
+
+### Parameters
+
+* `fpioa_func` for pin function
```
Comment (ConnorChristie) [inline en/libs/machine/network.md]: This doesn't match the parameters specified above

**Comment by Neutree** [comment]
Thanks~

**Pull request merged**